erpnext/translations/de.csv

```diff
@@ -8833,3 +8833,30 @@ Global Defaults,Allgemeine Voreinstellungen,
 Is Mandatory,Ist obligatorisch,
 WhatsApp,WhatsApp,
 Make a call,Einen Anruf tätigen,
+Enable Automatic Party Matching,Automatisches Zuordnen von Parteien aktivieren,
+Auto match and set the Party in Bank Transactions,"Partei automatisch anhand der Kontonummer bzw. IBAN zuordnen",
+Enable Fuzzy Matching,Unscharfe Zuordnung aktivieren,
```

Comment:
Sehr schwer zu verstehen.
„Fuzzy“ ist hier IMO verständlicher als „unscharf,“ und „Matching“ verständlicher als „Zuordnung.“
Auch wenn manch Traditionalist:in das ablehnen mag, würde ich hier tatsächlich auf „Fuzzy Matching“ zurückgreifen, worunter _jede:r_ unter 65 das Richtige versteht.